doc/nrf/protocols/matter/end_product/last_fabric_removal_delegate.rst

```diff
@@ -30,6 +30,21 @@ Predefined last fabric removal behaviors
 There are four predefined reactions to the last fabric removal available in the :ref:`matter_samples`.
 All behaviors are implemented as a delegation for the Fabric Table module, and the chosen reaction is run as a callback on each fabric removal.
 
+To enable the predefined behavior in your specific sample based on the :ref:`matter_samples` in the |NCS|, add the following lines to your :file:`app_task.cpp` project file:
+
+1. To the top of the :file:`fabric_table_delegate.h` header file:
```

Comment:
I don't get this point. Did you mean we should include `fabric_table_delegate.h` in the `app_task.cpp` or that we should add the code below to the `fabric_table_delegate.h`? If the first one, the example code is wrong.